lib/maze/retry_handler.rb

```diff
@@ -31,7 +31,11 @@ def should_retry?(test_case, event)
 
         if retry_on_driver_error?(event)
           $logger.warn "Retrying #{test_case.name} due to driver error: #{event.result.exception}"
-          Maze.driver.restart
+          if Maze.driver.is_a?(Maze::Driver::Appium) || Maze.driver.is_a?(Maze::Driver::ResilientAppium)
+            Maze.driver.restart
+          elsif Maze.driver.is_a?(Maze::Driver::Browser)
+            Maze.driver.refresh
```

Comment:
Does `Maze::Driver::Browser` have a `refresh` method? I don't see it or any `method_missing` shenanigans

Reply:
Yeah, added a passthrough for that

Reply:
Is it worth even calling it `restart`? I guess it's kind of misleading that way, but means the `is_a` checks aren't needed. I'm happy either way though

Reply:
I think it's worth being explicit that it's a refresh, no a restart, although I'm not opposed to changing it in the future